Admin/AbstractAdmin.php

```diff
@@ -3326,4 +3380,16 @@ private function buildRoutes()
             $extension->configureRoutes($this, $this->routes);
         }
     }
+
+    /**
+     * @internal
```

Comment:
I added a global NEXT_MAJOR on the `$cacheIsGranted` property. Should I dispatch the comment here and to magic methods too?